js/index.js

```diff
@@ -102,73 +57,16 @@ btnAdd.addEventListener('click', (e) => {
   form.reset();
 });
 
-// REMOVE BOOKS
-
-bookDisplay.addEventListener('click', (e) => {
-  e.preventDefault();
-  if (e.target.className === 'btn-remove') {
-    const { id } = e.target;
-    let books = LocalStore.getBooks();
-    books = books.filter((bk) => JSON.stringify(bk.id) !== id);
-    localStorage.setItem('local', JSON.stringify(books)); // LOCAL STORAGE
-    UI.deleteBookFromList(e.target);
-  }
-});
-
-// MENU NAVIGATION
-
-navList.addEventListener('click', () => {
-  page.style.display = 'block';
-  form.style.display = 'none';
-  contact.style.display = 'none';
-});
-
-navAdd.addEventListener('click', () => {
-  page.style.display = 'none';
-  form.style.display = 'block';
-  contact.style.display = 'none';
-});
+// GENERAL
 
-navContact.addEventListener('click', () => {
-  page.style.display = 'none';
-  form.style.display = 'none';
-  contact.style.display = 'block';
-});
+window.addEventListener('DOMContentLoaded', UI.showBooks);
+window.addEventListener('DOMContentLoaded', menuNav);
 
 // TIME DISPLAY
-function timeDisplay() {
-  const myDate = new Date();
-
-  const daysList = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
-  const monthsList = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'August', 'October', 'November', 'December'];
 
-  const date = myDate.getDate();
-  const month = monthsList[myDate.getMonth()];
-  const year = myDate.getFullYear();
-  const day = daysList[myDate.getDay()];
-
-  const today = `${day}, ${month} ${date}, ${year},`;
-
-  let amOrPm;
-  const twelveHours = () => {
-    if (myDate.getHours() > 12) {
-      amOrPm = 'PM';
-      const twentyFourHourTime = myDate.getHours();
-      const conversion = twentyFourHourTime - 12;
-      return `${conversion}`;
-    }
-    amOrPm = 'AM';
-    return `${myDate.getHours()}`;
-  };
-  const hours = twelveHours();
-  const minutes = myDate.getMinutes();
-
-  const currentTime = `${hours}:${minutes} ${amOrPm}`;
-
-  timeInfo.innerHTML = `${today} ${currentTime}`;
-}
-
-// GENERAL
+const timeDisplay = () => {
+  const timeInfo = document.querySelector('#time-info');
+  timeInfo.innerHTML = `${DateTime.now().toLocaleString(DateTime.DATETIME_MED)}`;
+};
```

Comment:
- [ ] This function is not working. Kindly take a second look. You can use += instead of = alone. Just a suggestion.